internal/pkg/print/print.go

```diff
@@ -15,6 +15,8 @@ const (
 	WithAggregateTypeDefaults
 	// WithRequired prints if inputs are required
 	WithRequired
+	// WithSortByRequired prints with required fields sorted to the top
```

Comment:
We could be more precise here. Example: "WithSortByRequired prints inputs sorted by "required", then by name.